call_your_mom/templates/call_your_mom/base.html

```diff
@@ -4,31 +4,61 @@
   <meta charset="UTF-8">
   <title>{% block page_title %}Call Your Mom{% endblock %}</title>
   <link rel="stylesheet" media="all" href="{% static 'main.css' %}"/>
+  <link href="https://maxcdn.bootstrapcdn.com/bootswatch/4.0.0/yeti/bootstrap.min.css" rel="stylesheet" integrity="sha384-ai4W/dNRpZ8Bl2+MNe2GXtf7urjWG5od4NHaCWEwrMeLzfDEQWZjFFIjQ6zG4R3W" crossorigin="anonymous">
+  <script defer src="https://use.fontawesome.com/releases/v5.0.9/js/all.js" integrity="sha384-8iPTk2s/jMVj81dnzb/iFR2sdA7u06vHJyyLlAd4snFpCl/SnyUjRrbdJsw1pGIl" crossorigin="anonymous"></script>
 </head>
 <body>
+<div class="container">
+  <div class="page-header">
+    <h1><i class="fas fa-phone-volume"></i> CALL! YOUR! MOM!</h1>      
+    <hr/>
+  </div>
+
   {% if messages %}
-  <ul>
     {% for message in messages %}
-    <li{% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message }}</li>
+		 <div class="alert alert-dismissible alert-{{ message.tags }}"><button type="button" class="close" data-dismiss="alert">&times;</button><strong>{{ message }}</strong></div>
     {% endfor %}
-  </ul>
   {% endif %}
 
   {% block content %}{% endblock %}
 
-  <footer style="color:silver; font-size: 80%;">{% with project_url="https://github.com/remram44/call_your_mom" %}{% blocktrans %}Call Your Mom sources available on <a href="{{ project_url }}">github.com</a>{% endblocktrans %}{% endwith %}<br/>
-
+  <footer>
+	<hr/>
+	{% with project_url="https://github.com/remram44/call_your_mom" %}{% blocktrans %}The source code for Call Your Mom is available via <a href="{{ project_url }}">github</a>. It is licensed under a modified BSD license, found in the <a href="https://github.com/remram44/call_your_mom/blob/master/LICENSE.txt">LICENSE.txt</a> file in the repository.
+  <br/>View this website in: 
+	{% endblocktrans %}{% endwith %}
   {% get_available_languages as LANGUAGES %}
   {% get_language_info_list for LANGUAGES as languages %}
   {% for language in languages %}
     {% if language.code == LANGUAGE_CODE %}
-      {{ language.name_local }}
+      <span class="badge badge-primary">{{ language.name_local }}</span>
     {% else %}
-      <a href="{% url 'set_lang' language.code %}">
+      <span class="badge badge-secondary">
+				<a href="{% url 'set_lang' language.code %}">
           {{ language.name_local }}
-      </a>
+      	</a>
+			</span>
     {% endif %}
   {% endfor %}
+	</ul>
```

Comment:
This doesn't seem to close anything?

Reply:
Whoops, that was a relic from a time when that was an inline list. Should be deleted.